.gitignore

```diff
@@ -42,3 +44,7 @@ auto-save-list
 
 # log
 *.log
+
+.idea/
+
+dispatches/models/renewables_case/.ipynb_checkpoints/
```

Comment:
I'm not sure what this does. Could you please double check with someone else if we can push this to the repo or not?

Reply:
I think this is ok.  We might even want to change the 2nd line here to `.ipynb_checkpoints/` so it picks up all of them.